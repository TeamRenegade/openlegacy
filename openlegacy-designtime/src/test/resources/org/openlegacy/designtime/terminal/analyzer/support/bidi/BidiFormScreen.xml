<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<snapshot type="INCOMING" sequence="10">
    <row rowNumber="1">
        <field value="מסך בעברית" column="40" />
    </row>
    <row rowNumber="4">
        <field value="אבגד  " editable="true" column="13" />
        <field value="שדה א:" column="23" />
    </row>
    <row rowNumber="5">
        <field value="בגדה  " column="13" />
        <field value="שדה ב:" column="20" />
    </row>
    <row rowNumber="6">
        <field value="1=כן 2=לא" visualValue="(אל=2 ןכ=1)" column="10" />
        <field value="1" visualValue="1" column="23" editable="true"/>
        <field value="שדה ג:" visualValue=":ג הדש" column="30" />
    </row>
    <size rows="24" columns="80"/>
</snapshot>
