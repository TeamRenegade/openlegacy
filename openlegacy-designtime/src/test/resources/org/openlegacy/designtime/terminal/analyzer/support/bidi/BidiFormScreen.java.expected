package com.test;

import org.openlegacy.annotations.screen.*;
import org.openlegacy.definitions.EnumGetValue;  

@ScreenEntity(displayName="מסך בעברית")
@ScreenIdentifiers(identifiers = { 
				@Identifier(row = 1, column = 40, value = "מסך בעברית"), 
				@Identifier(row = 4, column = 23, value = "שדה א:"), 
				@Identifier(row = 5, column = 20, value = "שדה ב:") 
				})
public class MscBabrit {
    
	
	@ScreenField(row = 4, column = 13, endColumn= 18 ,labelColumn= 23 ,editable= true ,displayName = "שדה א", sampleValue = "אבגד")
    private String sdeA;
	
	@ScreenField(row = 5, column = 13, endColumn= 18 ,labelColumn= 20 ,displayName = "שדה ב", sampleValue = "בגדה")
    private String sdeB;
	
	@ScreenField(row = 6, column = 23, endColumn= 23 ,labelColumn= 30 ,editable= true ,displayName = "שדה ג", sampleValue = "1")
    private SdeG sdeG;

    


 
	public enum SdeG implements EnumGetValue{
			Cn("1","כן"),
			La("2","לא")
		
		private String value;
		private String display;

		SdeG(String value, String display) {
			this.value = value;
			this.display = display;
		}

		public String getValue() {
			return value;
		}

		@Override
		public String toString() {
			return display;
		}
	}
}
